src/plugins/undoRedo/test/UndoRedo.e2e.js

```diff
@@ -167,6 +167,51 @@ describe('UndoRedo', () => {
           expect(getDataAtCell(3, 1)).toEqual('B4');
         });
 
+        it('should undo removal of single row after column sorting', () => {
```

Comment:
In my opinion, this test spec could be in `columnSorting.e2e.js`. What do you think?

Reply:
Since the undoRedo support was wrong (after doing columnSorting), it seems to me that the tests should be in the plugin undoRedo.